tests/test_requests.py

```diff
@@ -408,7 +408,7 @@ async def app(scope, receive, send):
 
     def post_body():
         yield b"foo"
-        yield "bar"
+        yield b"bar"
```

Comment:
`content` parameter doesn't accept both. I can undo this change if wanted.

Reply:
Nope - this is a very very sensible change. 😅

Reply:
Is this a "Nope" for the change itself, or for the "I can undo this change if wanted"? :thinking: 

If the first, I have this error: `E           TypeError: sequence item 1: expected a bytes-like object, str found`, and I don't think I can or should solve it on Starlette.

If the latter, ignore me. :sunglasses: :+1: